services/brig/Makefile

```diff
@@ -28,6 +28,7 @@ init:
 .PHONY: install
 install: init
 	stack install --pedantic --test --local-bin-path=dist
+	cp "$(shell stack path --dist-dir)/build/$(NAME)-integration/$(NAME)-integration" dist/
```

Comment:
bad rebase?

Reply:
Possibly, does this live in another branch?